light_curves/requirements_light_curve_generator.txt

```diff
@@ -14,6 +14,9 @@ astroquery>=0.4.8.dev0
 acstools
 lightkurve
 alerce
+lsdb
+s3fs
+nested_pandas
```

Comment:
no need to declare it as we don't directly use it (and lsdb should install it if it requires it)

Reply:
I removed nested_pandas from the requirements file.